src/components/statistics/Vaccines.js

```diff
@@ -1,5 +1,5 @@
-import axios from 'axios';
 import React, { useState, useEffect } from 'react';
+import axios from 'axios';

```

Comment:
o, średniki

Reply:
No to jest ciekawa niespójność, bo ty ogólnie ich nie lubisz i już je w niektórych miejscach wyciąłeś, ale defaultowo jak stworzysz apkę w reactcie to one wszędzie są. I defaultowo wszystkie prettify itp dodają je wszędzie.